src/main/resources/hapi-r4.properties

```diff
@@ -0,0 +1,151 @@
+# Adjust this to set the version of FHIR supported by this server. See
+# FhirVersionEnum for a list of available constants. Example values include
+# DSTU2, DSTU3, R4.
+fhir_version=R4
+
+# This is the address that the FHIR server will report as its own address.
+# If this server will be deployed (for example) to an internet accessible
+# server, put the DNS name of that server here.
+#
+# Note that this is also the address that the hapi-fhir-testpage-overlay
+# (the web UI similar to the one at http://hapi.fhir.org) will use to
+# connect internally to the FHIR server, so this also needs to be a name
+# accessible from the server itself.
+server_address=http://localhost:8080/hapi-fhir-jpaserver/fhir/
```

Comment:
This value will be different for each server - e.g., https://fhir4-dev.elimuinformatics.com